src/api/main/SignupForm.js

```diff
@@ -203,7 +203,19 @@ function runCaptcha(mailAddress: string, isBusinessUse: boolean, isPaidSubscript
 					}
 					const okAction = () => {
 						let captchaTime = captchaInput.value().trim()
-						if (captchaTime.match(/^[0-2][0-9]:[0-5][05]$/) && Number(captchaTime.substr(0, 2)) < 24) {
+						if (captchaTime.match(/^([0-2])?[0-9]:[0-5][05]$/) && Number(captchaTime.substr(0, 2)) < 24) {
+							let string_length = captchaTime.length;
+							if (captchaTime.indexOf(':') == 1) {
+								captchaTime = '0' + captchaTime;
+							} 
+							if ((Number(captchaTime.substr(0,2))) >= 12 && (Number(captchaTime.substr(0,2))) < 22) {
+								let hours = Number(captchaTime.substr(0,2))-12;
+								captchaTime = "0"+ hours + captchaTime.substr(2,string_length);
+								} else if (Number(captchaTime.substr(0,2)) > 21 && Number(captchaTime.substr(0,2)) <= 23) {
+									let hours = Number(captchaTime.substr(0,2))-12;
+									captchaTime = hours + captchaTime.substr(2,string_length);
+								} else if (Number(captchaTime.substr(0,2)) < 12) {}
+							
```

Comment:
Thank you!

But this code is quite hard to read and i think a bit too complex. I had a quick take at it that i find easier to read and it does less string manipulations. Feel free to comment

```javascript
// Make numbers from both tokens
[h, m] = captchaTime.split(':').map(t => Number(t))
// Cut numbers to hour and minute format, left pad with zeros, and glue with a colon
captchaTime = [h % 12, m % 60].map(a => String(a).padStart(2,'0')).join(':')
```

Reply:
That's a concise solution. I like it very much. :+1:

Reply:
Cool, i made a pull request with my variant.
But still: Thank you for your contribution! :+1: